### DISCUSSION TITLE
Cannot declare class TicketApiController, because the name is already in use

### DISCUSSION THREAD

**Discussion by abdo99ibrahim** [osTicket/osTicket#6468] [Q&A]
Good morning, when creating a ticket with API's getting this error 
[15-Feb-2023 13:17:11 UTC] PHP Fatal error:  Cannot declare class TicketApiController, because the name is already in use in C:\inetpub\wwwroot\osticket\osTicket-v1.12.5\upload\include\api.tickets.php on line 6

and when hashed this line the  #APIs work well

![2023-02-15_153614](https://user-images.githubusercontent.com/58654191/219042220-07c4c241-114a-45bd-81f2-c73f8021b08a.png)

**Comment by JediKev** [comment]
@abdo99ibrahim 

It appears you are using a really old version that is no longer supported (v1.12.5). I would highly recommend upgrading to one of the latest, supported releases and retesting. Also, you should use [the Forum](https://forum.osticket.com) for stuff like this in the future.

Cheers.
